app/domain/references/models/projections.py

```diff
@@ -94,21 +109,41 @@ def get_from_reference(
         )
 
     @classmethod
-    def _priority_sorted_enhancements(
+    def __priority_sorted_enhancements(
         cls, canonical_id: uuid.UUID, enhancements: list[Enhancement] | None
     ) -> list[Enhancement]:
         """
-        Order a references enhancements by priority for projecting.
+        Order a references enhancements by priority for projecting in reverse order.
```

Comment:
```suggestion
        Order a references enhancements by priority for projecting in increasing order.
```
I feel this makes it a little more clear?

Reply:
It most certainly does, thanks for suggesting!